packages/torch/torch.v0.16.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
```

Comment:
@LaurentMazare Hi sorry to bother you. Could you confirm ocaml-torch is changing maintainer? I'm not seeing any mentioned of this change in the original repository https://github.com/LaurentMazare/ocaml-torch but janestreet maintainers told me the change was agreed.

Reply:
@kit-ty-kate thanks for checking. Indeed this change was agreed and I can confirm that Jane Street will maintain the ocaml-torch package going forward, I'll tweak the readme of my repo to make this explicit.